### PR TITLE
UI Added support for User's profile image everywhere

### LINKED ISSUE #4543: UI: Show profile picture of users in Activity Feeds
We should display the profile picture of users in Activity Feeds and Entity pages for ownership

### PR DESCRIPTION
**Pull request by darth-coder00**
Closes #4543 
### Describe your changes :

I worked on the adding support for profile images on all pages

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">
Screenshot 2022-05-11 at 2 06 31 AM
Screenshot 2022-05-11 at 2 11 45 AM
Screenshot 2022-05-11 at 2 12 18 AM
Screenshot 2022-05-11 at 2 13 17 AM
Screenshot 2022-05-11 at 2 35 56 AM
Screenshot 2022-05-11 at 2 40 39 AM

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/FeedCardHeader/FeedCardHeader.tsx [hunk 1]
```diff
@@ -256,7 +256,7 @@ const FeedCardHeader: FC<FeedHeaderProp> = ({
           className="tw-cursor-pointer"
           data-testid="authorAvatar"
           onClick={onClickHandler}>
-          <Avatar name={createdBy} type="square" width="30" />
+          <ProfilePicture id="" name={createdBy} width="30" />
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/FeedCardHeader/FeedCardHeader.tsx:259]: same here

File: openmetadata-ui/src/main/resources/ui/src/utils/EntityUtils.tsx [hunk 2]
```diff
@@ -382,9 +382,10 @@ export const getInfoElements = (data: ExtraInfo) => {
           displayVal && displayVal !== '--' ? (
             isString(displayVal) ? (
               <div className="tw-inline-block tw-mr-2">
-                <Avatar
-                  name={displayVal}
-                  textClass="tw-text-xs"
+                <ProfilePicture
+                  displayName={displayVal}
+                  id=""
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/utils/EntityUtils.tsx:387]: same here for ID

File: openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/FeedCardHeader/FeedCardHeader.tsx [hunk 3]
```diff
@@ -92,7 +92,7 @@ const FeedCardHeader: FC<FeedHeaderProp> = ({
               <div>
                 <div className="tw-flex">
                   <div className="tw-mr-2">
-                    <Avatar name={createdBy} type="square" width="30" />
+                    <ProfilePicture id="" name={createdBy} width="30" />
```
[the remark this answers is not in the document]
Comment (darth-coder00) [reply openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/FeedCardHeader/FeedCardHeader.tsx:95]: `id` can be `""` but not undefined as per generated typescript, so cannot remove it.

**Review by ShaileshParmar11 — APPROVED** [review]
LGTM!

**Review by vivekratnavel — APPROVED** [review]

**Pull request merged**